### DISCUSSION TITLE
Allow multiple pick in Enumeration field

### DISCUSSION THREAD

**Discussion by stanleyyuenyiu** [strapi/strapi#7106] [Legacy Topics]
Enumeration field only able to pick one

it is not useful if not allow multiple

and you will need to create two database, then use many to many relationship to achieve this, which is quite stupid

**Comment by akhilmhdh** [comment]
Hi @stanleyyuenyiu Enum type is actually used in Graphql. But its just used generally throughout the rest too so that later migration to graphql wont be havoc. Enum is a graphql feature and all DB wont take it that easily.

 Attaching an old discussion which i came across on ENUM: This would give u an idea (https://github.com/strapi/strapi/discussions/4780)

About graphql enum : [Graphql Enum](https://blog.logrocket.com/what-you-need-to-know-about-graphql-enums/)

**Reply by derrickmehaffy** [reply]
Enumeration is actually a common term in many different types of languages (in fact most `boolean` values are technically enumeration values): https://en.wikipedia.org/wiki/Enumerated_type

@stanleyyuenyiu due to how enums function at a program language level, it is not really possible (or best practice) to allow for multiple enum values to be chosen. As you mentioned, currently the recommended way would be to use a relation (manyWay, far right of the relation menu).

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.
